tutorials/ar-subs/index.md

```diff
@@ -0,0 +1,795 @@
+1.  Optionally, if you use Ethernet connectivity, connect your ethernet cable to
+    the Pi.
+1.  Connect the Pi power supply to the wall socket and to the Pi, to boot up
+    Raspbian OS.
+1.  Login to the Pi with the initial username: `pi` and password: `raspberry`
+1.  Execute raspi-config as the superuser with:
+
+        sudo raspi-config
+
+1.  Change the pi user's password by selecting: **1 Change User Password**
+1.  The recommended network connection method is an ethernet cable which
+    provides a DHCP client IP to the Raspberry Pi. If you are using Wi-fi,
+    select: **2 Network Options** then **N2 Wi-fi** to
+    configure your Wi-fi country settings, SSID and passphrase.
+1.  To get most console apps to display text correctly, select:
+    **4 Localisation options** then **I1 Change Locale**. Scroll down with the
+    down arrow key, and deselect the default option (`en_GB.UTF-8 UTF-8`) using
+    the space bar.
+1.  Scroll down to `en_US.UTF-8 UTF-8` and select that option with the space
+    bar.
+1.  Press Tabulator and select: **Ok**
+1.  Use the arrow keys in the **Configuring locales** window to select
+    `en_US.UTF-8` from the list of choices as the default locale. Press
+    Tabulator and select: **Ok**
+1.  To set your system timezone, select: **4 Localisation options** then
+    **I2 Change Timezone** and use the menu options to find your geographic
+    region and country. Setting the timezone is important because some Google
+    Cloud services require client devices' system clocks to be within 10
+    minutes of real time. Raspbian uses global NTP time servers to synchronize
+    the clock.
+1.  Optionally, select: **4 Localisation options** then
+    **I3 Change Keyboard Layout** to match your layout. If you are not sure,
+    select: **Generic 105-key (Intl) PC**, then **English (US)**, then **The
+    default for the keyboard layout**, then **No compose key**.
+1.  Select: **Interfacing Options**, then **P2 SSH**, then **Yes** to enable the
+    SSH server on your Pi.
+1.  Select: **Finish**, to exit the raspi-config tool and accept to restart the
+    Pi. If you are not prompted to reboot when you exit raspi-config, reboot the
+    Pi by executing:
+
+        sudo reboot
+
+1.  Log in as the `pi` user with your new custom password.
+1.  Verify that you have an IP address on your Pi by executing:
+
+        ifconfig -a
+
+    You should see a client IP from your local network segment in either the
+    `wlan0` or `eth0` interface.
+
+1.  Verify that your DNS resolution and internet connection work, by executing:
+
+        ping www.google.com
+
+1.  If you do not have an IP address, or DNS resolution or internet connectivity
+    fail, do not proceed further until you have fixed your Pi's network
+    configuration.
+1.  For Wi-fi connectivity only - check if your Wi-fi adapter has power saving
+    enabled. We have to disable Wi-fi power saving, or it will give you sporadic
```

Comment:
Use 'you' instead of 'we' to address the reader, as you do in the rest of the tutorial

Reply:
Done